The `nix-review` output only shows the last part of the issue. It starts:

```bash
[...]
Running phase: checkPhase
ok      github.com/auth0/auth0-cli/internal/analytics   0.005s
ok      github.com/auth0/auth0-cli/internal/ansi        0.005s
ok      github.com/auth0/auth0-cli/internal/auth        20.044s
ok      github.com/auth0/auth0-cli/internal/auth/authutil       3.057s
ok      github.com/auth0/auth0-cli/internal/buildinfo   0.003s
Error: failed to deploy action with ID "1221c74c-cfd6-40db-af13-7bc9bb1c38db": 400 Bad Request
Usage:
  deploy [flags]

Examples:
  auth0 actions deploy
[...]
Flags:
  -h, --help              help for show
  -l, --language string   Language of the custom text. (default "en")

--- FAIL: TestFetchUniversalLoginBrandingData (0.01s)
    --- FAIL: TestFetchUniversalLoginBrandingData/it_can_correctly_fetch_universal_login_branding_data (0.00s)
        universal_login_customize_test.go:958: 
[...]
```

At first glance I would says that the test want to perform a "real" web requests. As the tests are passing locally according to your analysis, I guess that this test is not mocked.

The quick solution would be to skip the failing test. The sandbox has no network access.

```nix
  preCheck = ''
    [...]
    # Tests requires network access
    substituteInPlace internal/cli/universal_login_customize_test.go \
      --replace-fail "TestFetchUniversalLoginBrandingData" "SkipFetchUniversalLoginBrandingData"
  '';
```

--- a/pkgs/tools/admin/auth0-cli/default.nix
+++ b/pkgs/tools/admin/auth0-cli/default.nix
@@ -10,16 +10,17 @@ buildGoModule rec {
   src = fetchFromGitHub {
     owner = "auth0";
     repo = "auth0-cli";
-    rev = "v${version}";
+    rev = "refs/tags/v${version}";
     hash = "sha256-j7HT57/4rrrVkx9Zz+XuWD6UL0spdej+U5gWtFo1FSI=";
   };
 
   vendorHash = "sha256-bWAneCRsQbPRxEM/3jr1/Lov6NV67MRycOgrlj3bKF8=";
 
   ldflags = [
-    "-s" "-w"
-    "-X github.com/auth0/auth0-cli/internal/buildinfo.Version=v${version}"
-    "-X github.com/auth0/auth0-cli/internal/buildinfo.Revision=0000000"
+    "-s"
+    "-w"
+    "-X=github.com/auth0/auth0-cli/internal/buildinfo.Version=v${version}"
+    "-X=github.com/auth0/auth0-cli/internal/buildinfo.Revision=0000000"
   ];
 
   preCheck = ''

--- a/pkgs/tools/admin/auth0-cli/default.nix
+++ b/pkgs/tools/admin/auth0-cli/default.nix
@@ -28,6 +28,9 @@ buildGoModule rec {
     # This is because subPackages above limits what is built to just what we
     # want but also limits the tests
     unset subPackages
+    # Test requires network access
+    substituteInPlace internal/cli/universal_login_customize_test.go \
+      --replace-fail "TestFetchUniversalLoginBrandingData" "SkipFetchUniversalLoginBrandingData"
   '';
 
   subPackages = [ "cmd/auth0" ];

--- a/pkgs/tools/admin/auth0-cli/default.nix
+++ b/pkgs/tools/admin/auth0-cli/default.nix
@@ -5,16 +5,16 @@
 
 buildGoModule rec {
   pname = "auth0-cli";
-  version = "1.1.2";
+  version = "1.4.0";
 
   src = fetchFromGitHub {
     owner = "auth0";
     repo = "auth0-cli";
     rev = "v${version}";
-    hash = "sha256-EJH+Vn7wvrQ2umjmSXHjbgf2uf/kRbDoo0usTMqDFo4=";
+    hash = "sha256-j7HT57/4rrrVkx9Zz+XuWD6UL0spdej+U5gWtFo1FSI=";
   };
 
-  vendorHash = "sha256-T8y7MPFebDU6skfz4Rqo0ElRRaldtfexOl99D7h+orU=";
+  vendorHash = "sha256-bWAneCRsQbPRxEM/3jr1/Lov6NV67MRycOgrlj3bKF8=";
 
   ldflags = [
     "-s" "-w"

